api/support/verify_state.py

```
@@ -0,0 +1,265 @@
+
+
+async def _verify_chain(
+    session: aiohttp.ClientSession,
+    chain: str,
+    state_data: dict[str, Any],
+) -> list[str]:
+    """Run one verifier round for a chain. Returns Influx-format lines to emit."""
+    chain_lower = chain.lower()
+    chain_state = state_data.get(chain_lower)
+    if not chain_state or not chain_state.get("old_block"):
+        logging.warning(f"verify_state: no old_block for {chain}, skipping")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)